lib/src/repository.rs

```diff
@@ -41,6 +41,28 @@ pub trait RepositoryExt {
     /// This function returns all known issues known to the DIT repo.
     ///
     fn get_all_issue_hashes(&self) -> Result<HeadRefsToIssuesIter>;
+
+    /// Create a new message
+    ///
+    /// This function creates a new issue message as well as an appropriate
+    /// reference and return the new issue's hash. If the issue supplied is
+    /// none, the message will be the initial message of a new issue and the
+    /// hash returned will be also the hash of the new issue.
```

Comment:
Sorry, but this is not clear for me. Can your reword the documentation for this function, please?